index.html

```diff
@@ -8,14 +8,21 @@
       href="./assets/img/icons8-plane-24.png"
     />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./src/styles.scss" />
+    <!-- <link rel="stylesheet" href="./src/styles.scss" /> -->
+    <link href="./dist/styles.css" rel="stylesheet" />
+    <link href="./src/styles.css" rel="stylesheet" />
```

Comment:
Are both sytlesheets required?

Reply:
I have removed the comment and the extra style sheet link